Sources/Turf/CoreLocation.swift

```diff
@@ -54,12 +54,12 @@ public struct LocationCoordinate2D {
     /**
      The latitude in degrees.
      */
-    public let latitude: LocationDegrees
+    public internal(set) var latitude: LocationDegrees
```

Comment:
It looks like CLLocationCoordinate2D declares these properties as variable even for their getters, so ideally this Linux-specific declaration would be consistent with it.

Reply:
Good point. Done.